devices/assets/agent-src/build.sh

```
@@ -0,0 +1,30 @@
+echo "BUILD_TOOLS=$BUILD_TOOLS"
+
+rm -rf "$HERE/out"
+mkdir -p "$HERE/out/classes"
+
+echo "==> javac"
+javac -source 11 -target 11 -cp "$PLATFORM" -d "$HERE/out/classes" \
+  "$HERE/src/dev/mobilewright/agent/Agent.java"
+
+echo "==> d8"
+"$BUILD_TOOLS/d8" --min-api 24 --output "$HERE/out" --lib "$PLATFORM" \
+  $(find "$HERE/out/classes" -name '*.class')
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)